pkgs/by-name/gt/gtkclipblock/package.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, meson
+, cmake
+, ninja
+, pkg-config
+, gtk2
+, gtk3
+, gtk4
+}:
+
+let
+  distorm-src = fetchFromGitHub {
+    owner = "gdabah";
+    repo = "distorm";
+    rev = "3.5.2b";
+    hash = "sha256-2ftEV3TMS3HT7f96k+Pwt3Mm31fVEXcHpcbbz05jycU=";
+  };
+  source = fetchFromGitHub {
```

I believe it conveys better your intention of "factoring the things out":

```suggestion
  pname = "gtkclipblock";
  version = "0.2.2";

  src = fetchFromGitHub {
```